pom.xml

```diff
@@ -86,6 +136,16 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.22.0</version>
+				<configuration>
+					<argLine>
+						--illegal-access=permit
```

Comment:
The build is failing with this setting using Java 1.8 Would you like to update Java version in pom.xml?